app/src/views/ResultPage.vue

```diff
@@ -164,7 +164,8 @@
                       {{post.title}}
                     </v-list-item-title>
                     <v-list-item-subtitle :set="distance = postDistance(post)">
-                      <strong>{{post.location}} <em v-if="distance">(in {{distance}})</em></strong> &mdash; {{post.task}}
+                      <strong>{{post.location}} <em v-if="distance">(in {{distance}})</em></strong> &mdash;
+                       <em class="hide-copyright" v-html="post.task"/>
```

Comment:
Why is the whole task description now italic?
The em tag should emphasise a single aspect like the distance in this case, not the text as a whole.

Reply:
Tried to test out reactions on how it would look. Considering your reaction i removed it